source/vir/src/ast.rs

```diff
@@ -355,6 +355,7 @@ pub enum InvAtomicity {
 #[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
 pub enum AssertQueryMode {
     NonLinear,
+    BitVector,
```

Comment:
The comment below that says `/// assert_by with smt.arith.nl=true` probably needs to be updated.

Reply:
Thanks, I will update the comment